README.md

```diff
@@ -55,6 +55,34 @@ No. The status bar server is blocked on devices. However, macOS includes the fac
 
 The best idea is to check [the source code](https://github.com/shinydevelopment/SimulatorStatusMagic/blob/master/SDStatusBarManager/SDStatusBarManager.m) which should get you started with how it works :)
 
+## Updating for new versions of iOS
+There's a general pattern for updating this project to support a new version of iOS, although this could change if Apple changes something in the future.
+
+### Prepare the new files.
+1. Copy the previous releases's `SDStatusBarOverriderPostXX_Y.{h,m}` files, and update them to the new version.
+2. Update SDStatusBarManager.m to refer to the new overrider if detecting the new operating system version.
+
+### Find the updated structs in the runtime headers. 
+
+ 1. Download the latest version of the [dsdump](https://github.com/DerekSelander/dsdump) tool.
+ 2. Run dsdump against the UIKitCore framework binary, to generate the private runtime headers.
+ ```
+ ./dsdump --objc -a x86_64 --verbose=5 /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore --defined > ~/Desktop/UIKitCore.txt
+ ```
+3. Find `UIStatusBarServerListener` in the output. 
+
+### Update the structs in the new overrider.
+
+There are two structs, StatusBarRawData and StatusBarOverrideData, that need to be updated. Each correspond to a line in the runtime header output for UIStatusBarServerListener. It should be pretty easy to figure out the mapping if you study it for a minute.
+
+1. Update StatusBarRawData and StatusBarOverrideData to match any changes to the structs in the runtime headers.
+
+### Check if it works
+That should be it!
```

Comment:
Does the Markdown parser handle a heading without a newline after? It probably does, but just in case.